src/olympia/addons/tests/test_commands.py

```diff
@@ -165,6 +165,8 @@ def test_approve_addons_approve_files_no_review_type():
     assert file_.reload().status == amo.STATUS_PUBLIC
 
 
+@pytest.mark.django_db
+@mock.patch('olympia.reviewers.utils.sign_file', lambda f: None)
```

Comment:
curious, does `lambda f: None` work better than setting `return_value`?

Reply:
The nice thing about the `lambda f: None` as second argument is that I don't have to pass the mock as argument. But I could use `autospec=True` instead, that would be more reliable.